### ISSUE TITLE
Vapoursynth?

### ISSUE THREAD

**Issue by Selur** [erazortt/DoViBaker#4]
Would be nice if this would be available in Vapoursynth too.
Thanks!

**Comment by erazortt** [comment]
Yeah, I guess it would be nice to have that. But I have never made plugins for that and I have a lot of other work to do. So I wouldn't hold my breath.
